pages/thc/[processRef]/pause.tsx

```diff
@@ -36,7 +36,8 @@ const PausePage: NextPage = () => {
   const address = residentData.result?.address.join(", ");
   const tenants = (residentData.result?.tenants || [])
```

Comment:
To wrap this in a helper function:
`helperFunction((residentData.result?.tenants || [])
    .map((tenant) => tenant.fullName)
    .map((tenant) => tenant.fullName))`

Reply:
without the .join and .replace parts? how do i then call then call the function back in the files?

Reply:
The helper function will be what does the joining and replacing. You'll need to export the function const from that helper file and then import it into this file, like we do with `useDatabase`